tf_agents/agents/dqn/examples/v2/train_eval.py

```diff
@@ -87,6 +89,9 @@ def train_eval(
     train_sequence_length=1,
     # Params for QNetwork
     fc_layer_params=(100,),
+    a_fc_layer_params=(50,),
+    v_fc_layer_params=(50,),
+    network_type='duel_q_network',
```

Comment:
let's not make this the default, leave as q_network

Reply:
Thanks for the review. I will modify it

Reply:
Done